whisk/template/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.repo_name }}/models/model.py

```diff
@@ -1,6 +1,6 @@
+import whisk
 import pickle
-from whisk.model_stub import ModelStub
-import {{cookiecutter.project_name}}
+from whisk.model_stub import ModelStub # Remove when updated with your model
```

Comment:
I don't think we use `ModelStub` here in the template `Model` class - is this here as an example?

Reply:
Good catch. Removed.